docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -0,0 +1,61 @@
+---
+id: build-automation
+title: Automate plugin builds
+sidebar_position: 1
+description: Automate Grafana plugin builds and releases
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - automation
+  - build
+  - automate
+  - builds
+---
+
+# Automate packaging and signing with Github CI
+
+If your plugin has been set up to use the supplied [Github workflows](../create-a-plugin/develop-a-plugin/set-up-github-workflows.md) from [create-plugin](../get-started/get-started.mdx),
+the plugin should be built and packaged in the correct format.
+
+If you need to set up the Github worklows, see [these docs](https://github.com/grafana/plugin-actions/blob/main/build-plugin/README.md)
+
+We recommend using the ZIP file produced from this workflow to test the plugin.
```

Comment:
```suggestion
Additionally, we recommend using the zip file produced from this workflow to test the plugin.
```

Reply:
style note: doc team is in process of defining the style for "zip". i think this is the spelling that will probably be chosen. i think "zip" is what you had originally before my commit.